source/developer/developer-setup.md

```diff
@@ -139,6 +139,59 @@ Any issues? Please let us know on our forums at: http://forum.mattermost.org
 
 Any issues? Please let us know on our forums at: http://forum.mattermost.org
 
+### Windows ###
+
+1. Install and setup Docker
+    1. If you are using Win10 Pro/Ent, you might want to use Docker for Windows
+        1. Install [Docker for Windows](https://docs.docker.com/docker-for-windows/)
+        2. Add the line `127.0.0.1 dockerhost` to `C:\Windows\System32\drivers\etc\hosts` using editor with administrator privilege
+    2. For other version or you prefer using VirtualBox, use Docker Toolbox
+        1. Install [Docker Toolbox](https://www.docker.com/products/docker-toolbox)
+        2. Run the `Docker Quickstart Terminal` and let it configure `default` machine
+        3. Get `default` IP by run `docker-machine ip default` in the terminal
+        4. Add the line `<Docker-IP> dockerhost` to `C:\Windows\System32\drivers\etc\hosts` using editor with administrator privilege
+2. Install and setup Node.Js
+    1. Download and install from [https://nodejs.org/](https://nodejs.org/)
+    2. Open `Node.js command prompt` and run `npm config set cache <short-path>` to set npm global cache to shorter path, the default path will likely cause path over 255 characters.
```

Comment:
I forgot all about this @chikei.  Babun comes pre-packaged with a git client that interferes with the windows git client.  Just have to uninstall git from babun.

I think `pact remove git` will do the trick.  Im not a machine w/ babun install to test it.